build/configs/rtl8720e/loadable_apps/defconfig

```diff
@@ -214,7 +214,7 @@ CONFIG_BOOT_RUNFROMFLASH=y
 #
 # Boot Memory Configuration
 #
-CONFIG_RAM_KREGIONx_START="0x20080020,0x60000020,0x60100020,"
+CONFIG_RAM_KREGIONx_START="0x20080020,0x60000020,0x60101020,"
```

Comment:
Could you please explain why this change is needed? 0x60000020 + 1048576 = 0x60100020 so why does overlap happen? And if overlap happens should we change in 8721 also?

https://github.com/Samsung/TizenRT/blob/253b47dfbf3ea1f04f55bb4ff34d4296af6a57a8/build/configs/rtl8721csm/loadable_apps/defconfig#L212-L213